higher_health/views.py

```diff
@@ -1,18 +1,33 @@
 from django.shortcuts import render
 
-from .forms import LocationCheckerForm, LoginForm, PersonalInfoForm
+from django.http import HttpResponseRedirect
+from django.core.exceptions import ValidationError
+from .forms import HealthCheckQuestionnaire, HealthCheckLogin
 
 
-def healthcheck_location(request):
-    form = LocationCheckerForm()
-    return render(request, "health-check_location-questionnaire.html", {"form": form})
 
+def healthcheck_questionnaire(request):
+    submitted = False
+    if request.method == 'POST':
+        form = HealthCheckQuestionnaire(request.POST)
+        if form.is_valid():
+            cd = form.cleaned_data
+            # assert False
+            return HttpResponseRedirect('/?submitted=True')
```

Comment:
We'll have to do something more than just a query parameter for the receipt, but I think that this is good just to see how the UI/UX is, I'll replace it with proper backend lookup and validation before we give them a receipt.

Reply:
@rudigiesler Sounds great. Thank you!